### ISSUE TITLE
This project looks py3k compliant. Your setup.py don't say it is.

### ISSUE THREAD

**Issue by Christophe31** [johnsensible/django-sendfile#25]
I'm asking my-self if I make one of my projects python 3.4/django 1.7 as I have big refactoring in front of me.

I used the nice tool https://caniusepython3.com/ and it told me that django-sendfile is not py3k compliant. When I read my self, it looks already compliant or trivial to port.

You probably should update your setup.py except if you haven't tried your project on py3k yet.    

If you ran your tests against python 3, you probably should add this to your classifiers:

classifiers = [
        "Programming Language :: Python :: 2",
        "Programming Language :: Python :: 3",]

**Comment by timabbott** [comment]
Yeah, it'd be great to have this tagged as supporting Python 3.  We're looking at using this project in Zulip and we're close to moving to full Python 3 support.  We'll likely report bugs if it doesn't work :)
